prime-react/src/main/scala/lucuma/react/primereact/DropdownBase.scala

```diff
@@ -65,6 +72,20 @@ object DropdownBase {
       .applyOrNot(props.disabled, _.disabled(_))
       .applyOrNot(props.dropdownIcon, _.dropdownIcon(_))
       .applyOrNot(props.tooltip, _.tooltip(_))
+      .applyOrNot(
+        props.itemTemplate,
+        (c, p) =>
+          c.itemTemplate(raw =>
+            p(props.selectItemFinder(raw.asInstanceOf[CSelectItem].value)).rawNode
+          )
+      )
+      .applyOrNot(
+        props.valueTemplate,
```

Comment:
Have you tested this for `DropdownOptional`? Seems like `selectItemFinder` might fail?

Reply:
Good catch, I added an extra param to support handling empty cases